editor.planx.uk/src/theme.ts

```diff
@@ -45,33 +72,11 @@ export const linkStyle = (primaryColor?: string) => ({
   "&:focus-visible": focusStyle,
 });
 
-/**
- * Get Global theme options
- * The global theme is used in editor, and as the base theme in Preview/Unpublished which can be
- * merged with Team specific options
- */
-export const getGlobalThemeOptions = (): ThemeOptions => {
+const getThemeOptions = (primaryColor: string): ThemeOptions => {
   const palette = createPalette({
```

Comment:
Here's the bug - calling this without the team's primary colour generates secondary colours (e.g. `theme.palette.primary.dark`) which are then never overwritten.